packages/admin/admin/src/index.ts

```diff
@@ -138,6 +138,16 @@ export { StackLink } from "./stack/StackLink";
 export { StackPageTitle } from "./stack/StackPageTitle";
 export { IStackSwitchApi, StackSwitch, StackSwitchApiContext, useStackSwitch, useStackSwitchApi } from "./stack/Switch";
 export { StackSwitchMeta } from "./stack/SwitchMeta";
+export {
+    SubmissionBoundary,
+    SubmissionBoundaryApi,
+    SubmissionBoundaryApiContext,
+    SubmissionBoundaryState,
+    SubmissionBoundaryStateProps,
```

Comment:
Do we need to export the props type?

Reply:
In many cases we export them, not in all though.

generally, it makes it easer to extend a component and forward it's props if the Props type is exported